internal/mediainfo/ac3.go

```
@@ -549,10 +549,14 @@ func parseAC3Frame(payload []byte) (ac3Info, int, bool) {
 	return info, frameSize, true
 }
 
+// parseEAC3Frame parses one E-AC-3 syncframe and scans that frame for JOC metadata.
 func parseEAC3Frame(payload []byte) (ac3Info, int, bool) {
```

golangci-lint (2.12.2)

[error] func parseEAC3Frame is unused

(unused)